dev-infra/commit-message/validate-range/validate-range.ts

```diff
@@ -32,7 +38,7 @@ export function validateCommitRange(range: string) {
       disallowSquash: true,
       nonFixupCommitHeaders: isNonFixup(commit) ?
           undefined :
-          commits.slice(0, i).filter(isNonFixup).map(extractCommitHeader)
+          commits.slice(i).filter(isNonFixup).map(extractCommitHeader)
```

Comment:
Should this be reverted? Seems like this is wrong and now capturing commits from `[i, ∞[` and no longer from the beginning to `i`.

Reply:
Previously, we were getting from the beginning of the list to the index. The array created by this method of extracting commit messages is reversed from how we did it before, so instead we now get the index to the end of the array.

We move from `slice(0, i)` to `slice(0, array.length -1)`

Reply:
Nit: You can start at `i + 1`:

```suggestion
          commits.slice(i + 1).filter(isNonFixup).map(extractCommitHeader)
```